### ISSUE TITLE
React.createClass is not available in React 16

### ISSUE THREAD

**Issue by paulmelnikow** [mathisonian/react-editable-svg-label#16]
As I mentioned in https://github.com/mathisonian/react-editable-svg-label/issues/6#issuecomment-442175882, `React.createClass` is deprecated.

The recommended solution is to use a regular class:
https://reactjs.org/blog/2017/04/07/react-v15.5.0.html#migrating-from-reactcreateclass

**Comment by paulmelnikow** [comment]
I think using a class will require a tooling change:

```
> react-editable-svg-label@1.1.1 build /Users/pnm/code/react-editable-svg-label
> gulp clean && NODE_ENV=production gulp build

[22:12:59] Using gulpfile ~/code/react-editable-svg-label/gulpfile.js
[22:12:59] Starting 'clean:dist'...
[22:12:59] Starting 'clean:lib'...
[22:12:59] Starting 'clean:examples'...
[22:12:59] Finished 'clean:lib' after 11 ms
[22:12:59] Finished 'clean:dist' after 16 ms
[22:12:59] Finished 'clean:examples' after 12 ms
[22:12:59] Starting 'clean'...
[22:12:59] Finished 'clean' after 5.41 μs
[22:13:00] Using gulpfile ~/code/react-editable-svg-label/gulpfile.js
[22:13:00] Starting 'build:dist:scripts'...
[22:13:00] Starting 'build:lib'...
[22:13:00] Starting 'build:example:files'...
[22:13:00] Starting 'build:example:css'...
[22:13:00] Starting 'build:example:scripts'...

events.js:160
      throw er; // Unhandled 'error' event
      ^
SyntaxError: /Users/pnm/code/react-editable-svg-label/src/ReactEditableSvgLabel.js: Unexpected token (7:12)
class ReactEditableSvgLabel extends React.Component {
```

So either #5 needs to be resolved first, or we could add `create-react-class` as a dependency.

**Comment by paulmelnikow** [comment]
Fixed via #14.
